.github/workflows/ci.yml

```diff
@@ -79,3 +79,14 @@ jobs:
           flags: "unittests-${{ matrix.python-version }}"
           name: codecov-umbrella
           verbose: true
+
+  build-and-test-image:
+    if: github.repository == 'ansible/ansible-rulebook'
```

Comment:
I think forks might want to run this as well, any contribution can break the image.

Reply:
yes, but under that logic we would prevent to run everything in forks. The purpose is be able to run the workflows in the fork prior the PR. IMO, only the workflows that depends on the repo (eg due to secrets) should be disabled. 
If some fork doesn't want to run a specific workflow they can disable it there.

Reply:
Fixed.